CHANGES.rst

```diff
@@ -155,6 +155,7 @@ regtest
 -------
 
 - Updated input filenames for NIRCam ``wfss_contam`` tests [#7595]
+- Updated NIRSpec IFU tests to use flight data [#7620]
```

Comment:
Need to move this entry to the top of change log, now that we've had a couple releases since this PR branch was constructed.

Reply:
Still need to move this entry to the top (unreleased section) of the change log.

Reply:
done